CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Fixed
 - PR template typo
 
+## [3.0.0] - 2018-02-09
```

Comment:
please remove any versioning we do this post acceptance. This is done for several reasons:
- PRs are not FIFO, this reduces the number of times we have to ask users to rebase
- maintainers might disagree on how the change impacts users
- we date the release not the date submitted

Also in this case the version is already `3.0.0`

Reply:
I think that sounds real good. I'm not sure how to add the logging, but something along the lines of "No pager_team found in handler config, using default api key." should be good enough.

Reply:
Just add something like this on the line before assignment:
```
puts "you configured your client to use a pager team of #{@event['client']['pager_team']} but it was nil, falling back to default key"
```